packages/platform-ios/src/link-pods/findMarkedLinesInPodfile.js

```diff
@@ -7,7 +7,7 @@
  * @format
  */
 
-const MARKER_TEXT = '# Add new pods below this line';
+export const MARKER_TEXT = '# Add new pods below this line';
```

Comment:
can we improve this text to include `DO NOT REMOVE`? I had no idea it's a marker for CLI

Reply:
Yes, but we need to handle both this and legacy one. Let's do that in a sep. ticket + some documentation on how this is set up.